### ISSUE TITLE
linkToDestination does not works with SVG

### ISSUE THREAD

**Issue by JulienLecoq** [bpampuch/pdfmake#2770]
It does works with a `png` like so:

```ts
{
  linkToDestination: 'destinationId',
  image: myIcon,
}
```

But does not with an `svg` like so:

```ts
{
  linkToDestination: 'destinationId',
  svg: myIcon,
}
```

Clicking on the svg will not do anything while clicking on the png will redirect the user to the element with id: 'destinationId'.

**Comment by liborm85** [comment]
Released in version 0.2.15.

**Comment by JulienLecoq** [comment]
Nice, I will be to update my PDF generation :D
